packages/snippet-bot/package.json

```diff
@@ -30,7 +30,11 @@
   },
   "dependencies": {
     "@octokit/rest": "^18.0.0",
+    "@types/minimatch": "^3.0.3",
     "gcf-utils": "^5.3.0",
+    "install": "^0.13.0",
```

Comment:
You can delete `npm` and `install` from the list here

Reply:
done